server/comment/hooks.js

```diff
@@ -39,7 +39,9 @@ async function sendEmailNotification(comment, db) {
   });
   const { author, repository, activity } = comment;
   const data = {
+    repositoryId: repository.id,
```

Comment:
By introducing `repositoryId`, I think it would be nicer to rename `repository` to `repositoryName` or pass the entire `repository` object. Thoughts 🤔?

Reply:
I thinks either approach is fine, but will go with renaming `repository ` to `repositoryName` since there is no need to pass the whole object. 🤷‍♂️